raiden/tests/integration/long_running/test_settlement.py

```diff
@@ -51,6 +53,27 @@ def wait_for_batch_unlock(app, token_network_address, receiver, sender):
         )
 
 
+def is_channel_registered(
+    node_app: App, partner_app: App, canonical_identifier: CanonicalIdentifier
+) -> bool:
+    """True if the `node_app` has a channel with `partner_app` in its state."""
+    token_network = views.get_token_network_by_address(
+        chain_state=views.state_from_app(node_app),
+        token_network_address=canonical_identifier.token_network_address,
+    )
+    assert token_network
```

Comment:
Is this `assert`  safe? Or should it be `if token_network is None: return False`?

Reply:
This is only used in the tests, it should be safe to have it as an assert.